privacy/whonix-customize.md

```diff
@@ -22,13 +22,21 @@ You will want to complete the following instructions in both the **Whonix-Gatewa
 
 ### Configuring Whonix-Gateway
 
-Launch the `dom0` terminal app `Konsole` from your Qubes App Launcher. Then get a list of current kernel parameters.
+Launch the `dom0` terminal app `Konsole (konsole)` or `Terminal emulator (xfce4-terminal)` from your Qubes App Launcher. Then get a list of current kernel parameters.
+
+In Qubes version 3 releases you have to do this with
 
 ~~~
 qvm-prefs -l whonix-gw kernelopts
```

Comment:
In both 3.2 and 4.0 `qvm-prefs -g whonix-gw kernelopts` will work here. And will also work `qvm-prefs whonix-gw kernelopts`.